Extensions/dnSpy.Debugger/dnSpy.Debugger/DbgUI/StartDebuggingOptionsProvider.cs

```diff
@@ -87,6 +89,9 @@ sealed class StartDebuggingOptionsProvider {
 
 			var oldOptions = mru.TryGetOptions(filename);
 			var lastOptions = mru.TryGetLastOptions();
+			if (oldOptions == null && lastOptions.HasValue && lastOptions!.Value.options != null && debuggerSettingsImpl.Value.DontAutoUpdateDebugLaunchSettings)
```

Comment:
`lastOptions!.Value.options != null` is redundant here as `options` element in the tuple is not marked as nullable and thus should never be `null`.